exo-go/src/logger/index_test.go

```diff
@@ -0,0 +1,14 @@
+package logger_test
+
+import (
+	"github.com/Originate/exosphere/exo-go/src/logger"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Logger", func() {
```

Comment:
I don't think we can test the logger particularly well, but let's test the logic that parses Docker Compose output.